### PR TITLE
GPTQ + Scale Eestimation conformance test

### PR DESCRIPTION
**Pull request by alexsu52**
### Changes

replaced tinyllama_data_aware_gptq test by tinyllama_data_aware_gptq_scale_estimation_stateful test in the weight compression conformance test

### Reason for changes

Test regression in the GPTQ + Scale Estimation pipeline

### Related tickets

ref 143637

### Tests

tinyllama_data_aware_gptq_scale_estimation_stateful
NNCF/job/manual/job/post_training_weight_compression/226

### CODE REVIEW

File: tests/post_training/data/wc_reference_data.yaml [hunk 1]
```diff
@@ -22,8 +22,8 @@ tinyllama_int8_data_free_backend_TORCH:
   metric_value: 0.95624
   num_int4: 0
   num_int8: 312
-tinyllama_data_aware_gptq_backend_OV:
-  metric_value: 0.87134
+tinyllama_data_aware_gptq_scale_estimation_stateful_backend_OV:
+  metric_value: 0.8326
```
Comment (ljaljushkin) [inline tests/post_training/data/wc_reference_data.yaml:26]: hmm, it was much better. Wonder why similarity doesn't reflect improvement in the method. 
did you have a chance to look at the generated output? Does it contain empty strings or repetitions?

**Comment by AlexanderDokuchaev** [comment]
@alexsu52  will you update this PR?

**Comment by alexsu52** [comment]
> @alexsu52 will you update this PR?

 job/manual/job/post_training_weight_compression/210/

File: tests/post_training/data/wc_reference_data.yaml (hunk 1 above)
Comment (alexsu52) [reply tests/post_training/data/wc_reference_data.yaml]: The issue is related to OpenVINO. Updated the metric.

**Comment by ljaljushkin** [comment]
> > @alexsu52 will you update this PR?
> 
> job/manual/job/post_training_weight_compression/210/

the job was aborted, which job is actual?

**Comment by alexsu52** [comment]
> > > @alexsu52 will you update this PR?
> > 
> > 
> > job/manual/job/post_training_weight_compression/210/
> 
> the job was aborted, which job is actual?

NNCF/job/manual/job/post_training_weight_compression/226

**Review by ljaljushkin — APPROVED** [review]

**Pull request merged**